tests/connectors/test_google_big_query.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+import pandas as pd
+from unittest.mock import Mock,patch
+from pandasai.connectors.base import GoogleBigQueryConnectorConfig
+from pandasai.connectors import GoogleBigQueryConnector
+
+class TestGoogleBigQueryConnector(unittest.TestCase):
```

Comment:
Are we on purpose using two testing frameworks (pytest and unittest)?

Reply:
Don't know about that . I wrote similar testcases as previously did in the case of `MySqlConnector` and other `sql connectors`.